src/middlewares/response.middleware.ts

```diff
@@ -0,0 +1,34 @@
+import { NextFunction, Request, Response } from 'express';
+import { dayjs } from '@utils/dayjs.util';
+
+export const responseMiddleware = () => {
+  return (body: Object, req: Request, res: Response) => {
+    const timestamp = dayjs().utc().format();
+
+    if (Array.isArray(body['items']) && body['pagination']) {
+      return res.json({
```

Comment:
We'd want to make this dynamic, it should return the status code of what was specified in the controller e.g `response.status(200)` or `response.status(201)`